moveit_ros/visualization/rviz_plugin_render_tools/src/trajectory_visualization.cpp

```diff
@@ -399,13 +399,17 @@ void TrajectoryVisualization::update(float wall_dt, float ros_dt)
     if (animating_path_)
     {
       current_state_ = -1;
-      current_state_time_ = std::numeric_limits<float>::infinity();
+      current_state_time_ = 0;
       display_path_robot_->update(displaying_trajectory_message_->getFirstWayPointPtr());
       display_path_robot_->setVisible(display_->isEnabled());
       if (trajectory_slider_panel_)
         trajectory_slider_panel_->setSliderPosition(0);
     }
   }
+  else
```

Comment:
add comment explaining why this is necessary? i.e.

    // if currently animating path, increment time